docs/source/blocks/blocks-core/teaser.md

```diff
@@ -0,0 +1,152 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Teaser Block description and developer notes"
+    "property=og:description": "Volto Teaser Block description and developer notes"
+    "property=og:title": "Volto Teaser Block"
+    "keywords": "Volto, React, blocks, teaser, Plone"
+---
+
+(teaser-block)=
+
+# Teaser Block
+
+Volto comes with a Teaser block that given a content object as target is able to pull information from it and show it in the block.
+By default, this information is the `title`, `description`, `preview_image` and shows them in a summary (Image on left, title, description) way.
+
+## Blocks settings
+
+The Teaser has the following settings:
+
+Target
+: The target is either an existing content item in your Plone site that can be selected by clicking the list icon, or an external URL that can be typed into the text field and saved by clicking the right arrow when it appears after you start typing.
```

Comment:
Anytime we describe interactions with the UI, we should provide a screenshot or video. Then this could be reused for the User Manual.

Reply:
I copied over the user manual without paying much attention that it was describing the UI... sorry about that. My intention is that since this is the developer guide, we only mention the settings. Let me fix it.